status/versions.go

```diff
@@ -27,12 +27,15 @@ var (
 	//   redhat@redhat-docker.io/maistra-0.1.0-1-3a136c90ec5e308f236e0d7ebb5c4c5e405217f4-unknown
 	// Example Maistra upstream project version is:
 	//   redhat@redhat-pulp.abc.xyz.redhat.com:8888/openshift-istio-tech-preview-0.1.0-1-3a136c90ec5e308f236e0d7ebb5c4c5e405217f4-Custom
+	// Example OpenShift Service Mesh 1.1 product version is:
+	//   OSSM_1.1.0-291c5419cf19d2b015e7e5dee970c458fb8f1982-Clean
```

Comment:
Do you also want to update Maistra (community version of OSSM) case?

It prints `Maistra` instead of `OSSM`. Example:
`Maistra_1.1.0-291c5419cf19d2b015e7e5dee970c458fb8f1982-Clean`

Reply:
Ah.. yes. if Maistra 1.1 has the new format, too, then yes. I'll do that.